recipe/activate.sh

```diff
@@ -23,10 +23,11 @@ else
     CUDA_LDFLAGS="${CUDA_LDFLAGS} -L${CONDA_PREFIX}/${targetsDir}/lib/stubs"
 fi
 
-export CFLAGS="${CFLAGS} ${CUDA_CFLAGS} ${CUDA_LDFLAGS}"
-export CPPFLAGS="${CPPFLAGS} ${CUDA_CFLAGS} ${CUDA_LDFLAGS}"
-export CXXFLAGS="${CXXFLAGS} ${CUDA_CFLAGS} ${CUDA_LDFLAGS}"
+export CFLAGS="${CFLAGS} ${CUDA_CFLAGS}"
+export CPPFLAGS="${CPPFLAGS} ${CUDA_CFLAGS}"
+export CXXFLAGS="${CXXFLAGS} ${CUDA_CFLAGS}"
 export LDFLAGS="${LDFLAGS} ${CUDA_LDFLAGS}"
+export CUDAFLAGS="${CUDAFLAGS} ${CUDA_CFLAGS}"
```

Comment:
I believe that the correct home for this is in nvcc.profile ( https://github.com/conda-forge/cuda-nvcc-impl-feedstock/blob/main/recipe/nvcc.profile.patch ) and not a CMake only env variable.

Reply:
I think it makes sense to do something that changes nvcc directly to support all build systems. Do you have a recent documentation for this file? I could only find something from 2011.

https://developer.download.nvidia.com/compute/DevZone/docs/html/C/doc/nvcc.pdf

Reply:
It doesn't but you can dereference env variables. So we can have branching logic in the activation script which saves to `CUDA_INCLUDE_DIRS` and `CUDA_LINK_DIRS` and have `nvcc.profile` use those variables